docs/asciidoc/modules/ROOT/partials/usage/apoc.load.directory.async.add.adoc

```diff
@@ -0,0 +1,57 @@
+The first parameter is the name of our custom watch listener.
+If I call a procedure with the same name as an existing one, this will be overwritten.
+The second parameter is the cypher query that will be executed.
+The query can have the following parameters:
+
+* $fileName: the name of the file which triggered the event
+TODO
+* $filePath: the absolute path of the file which triggered the event if `apoc.import.file.use_neo4j_config=false`, otherwise the relative path starting from ``$IMPORT_DIR`
+* $fileDirectory: the absolute path directory of the file which triggered the event if `apoc.import.file.use_neo4j_config=false`, otherwise the relative path starting from ``$IMPORT_DIR`
+* $listenEventType: the triggered event ("CREATE", "DELETE" or "MODIFY")
+
+
+The third parameter is the pattern of file to search for.
+By default is '*', that is, search all files.
+The fourth is the search path of directory.
+By default is an empty string, that is, search file in `import` directory.
+
+Let's suppose we have set the config option:
+----
+apoc.import.file.use_neo4j_config=true
+----
+
+We can do the following procedure:
+
+[source,cypher]
+----
+CALL apoc.load.directory.async.add('test', 'CREATE (n:Test)');
+----
+so every time, in `import` folder, a file is created/deleted/updated will be executed the query `CREATE (n:Test)`.
+
+We can listen only for specified event kinds, for example for file creation, and create an import report:
+----
+CALL apoc.load.directory.async.add('test', 'CREATE (n:Import {fileName: $fileName})', '*', '', {eventKinds: ['ENTRY_CREATE']});
```

Comment:
no Giuse dobbiamo contestualizzarlo con una load csv ad esempio e correggi le opzioni che sono cambiate

Reply:
qui e ovunque ci sono delle chiamate alle nostre procedure

Reply:
Sì, ho messo nel file `docs/asciidoc/modules/ROOT/pages/import/load-directory-async.adoc` le chiamate alle procedure load csv e load json, che viene richiamato in tutte le procedure, così da qualsiasi documentazione inerente la `apoc.load.directory.async.**` si può arrivare a quegli esempi. Questo file è specifico per la add, ma volendo posso mettere l'esempio con csv e json anche qui?

Reply:
@vga91 occhio non hai taggato me😂.
Cmq si mettilo in tutti gli esempi non vogliamo confondere la gente, deve essere chiaro l'utilizzo finale che ne deve essere fatto